scipy/integrate/_ivp/ivp.py

```diff
@@ -234,13 +234,14 @@ def solve_ivp(fun, t_span, y0, method='RK45', t_eval=None, dense_output=False,
     t_eval : array_like or None, optional
         Times at which to store the computed solution, must be sorted and lie
         within `t_span`. If None (default), use points selected by the solver.
-    events : callable, list of callables or None, optional
-        Types of events to track. Each is defined by a continuous function of
-        time and state that becomes zero value in case of an event. Each function
-        must have the signature ``event(t, y)`` and return a float. The solver will
-        find an accurate value of ``t`` at which ``event(t, y(t)) = 0`` using a
-        root-finding algorithm. Additionally each ``event`` function might have
-        the following attributes:
+    events : callable, or list of callables, optional
+        Types of events to track. If None (default), events won't be tracked.
+        Each is defined by a continuous function of time and state that becomes
+        zero value in case of an event. Each function must have the signature
+        ``event(t, y)`` and return a float. The solver will find an accurate
+        value of ``t`` at which ``event(t, y(t)) = 0`` using a root-finding
```

Comment:
Very minor issue, the double backticks around t, should only be a single backtick.